### PR TITLE
chore(deps): use udd

### PR DESCRIPTION
**Pull request by erfanium**

### CODE REVIEW

File: .github/workflows/udd.yml [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+    - cron: "42 8 * * *"
+
+jobs:
+  update-dependencies:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: denoland/setup-deno@v1
+        with:
+          deno-version: 1.15.3
```
Comment (lucsoft) [inline .github/workflows/udd.yml]: Could you change that to v1.x

File: .github/workflows/udd.yml [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+      - name: Update dependencies
+        run: |
+          deno run -A https://deno.land/x/udd/main.ts deps.ts test_deps.ts
+      - name: Create Pull Request
```
Comment (lucsoft) [inline .github/workflows/udd.yml:19]: Could we change that instead failing most of the time it will just skip all the steps and don't fail the action?

**Review by lucsoft — CHANGES_REQUESTED** [review]

File: .github/workflows/udd.yml (hunk 1 above)
Comment (erfanium) [reply .github/workflows/udd.yml]: Ok

File: .github/workflows/udd.yml (hunk 2 above)
Comment (erfanium) [reply .github/workflows/udd.yml:19]: I just copied this code from udd repo. it should works well

**Pull request merged**